motoman/045ab54/045ab54.bug

```diff
@@ -28,9 +28,9 @@ bug:
   reproducibility: always
   trace: N/A
 fix:
-  repo: https://github.com/JeremyZoss/motoman | https://github.com/ros-industrial/motoman
-  hash: null
-  pull-request: 63
+  repo: https://github.com/ros-industrial/motoman
+  hash: 045ab5403733ee9ec2512ee4d22e541dd3d6424f
+  pull-request: https://github.com/ros-industrial/motoman/pull/63
```

Comment:
I've used a full URL here, as I remembered we did that earlier.

But looking at the other `.bug`s, it seems we've changed to only list IDs?

Reply:
Nice catch. I've created an issue (#390). Let's make these consistent in a separate PR.